components/measure-overview.js

```diff
@@ -134,7 +128,8 @@ export default function MeasureOverview({ viewport, children, data, ...props })
           </div>
           <div className='flex items-center w-10/12'>
             <span className='overview-text border border-black rounded-xl px-2 uppercase'>
-              {data?.measure?.juridischHaalbaarheid}
+              {data?.measure?.juridischeHaalbaarheid}
+              {console.log(data?.measure?.juridischeHaalbaarheid)}
```

Comment:
@theocampbell left this console log intentionally?

Reply:
@gurdenbatra nope! just took it out and added another commit.